src/Runners/PHPUnit/Options.php

```diff
@@ -115,6 +156,18 @@ public function __get($var)
     }
 
     /**
+     * Public read accessibility
+     * (e.g. to make empty($options->property) work as expected)
+     *
+     * @param string $var
+     * @return mixed
+     */
+    public function __isset($var)
```

Comment:
If all properties are explicitly defined, is there still a need for magic methods?